documentation/src/main/asciidoc/userguide/chapters/caching/Caching.adoc

```diff
@@ -724,15 +836,36 @@ If the Infinispan `CacheManager` is bound to JNDI, then the `JndiInfinispanRegio
 ----
 ====
 
-===== Infinispan in JBoss AS/WildFly
+==== Inside Wildfly
 
+In WildFly, Infinispan is the default second level cache provider for JPA/Hibernate.
 When using JPA in WildFly, region factory is automatically set upon configuring `hibernate.cache.use_second_level_cache=true` (by default second-level cache is not used).
-For more information, please consult https://docs.jboss.org/author/display/WFLY9/JPA+Reference+Guide#JPAReferenceGuide-UsingtheInfinispansecondlevelcache[WildFly documentation].
+
+You can find details about its configuration in link:{wildflydocroot}/JPA%20Reference%20Guide[the JPA reference guide], in particular, in the link:{wildflydocroot}/JPA%20Reference%20Guide#JPAReferenceGuide-UsingtheInfinispansecondlevelcache[second level cache] section.
+
+The default second-level cache configurations used by Wildfly match the configurations explained above both for local and clustered environments.
+So, an Infinispan based second-level cache should behave exactly the same standalone and within containers that provide Infinispan second-level cache as default for JPA/Hibernate.
+
+[IMPORTANT]
+====
+Remember that if deploying to Wildfly or Application Server, the way some Infinispan second level cache provider configuration is defined changes slightly because the properties must include deployment and persistence information.
+Check the <<caching-provider-infinispan-config,Configuration>> section for more details.
+====
 
 [[caching-provider-infinispan-config]]
 ==== Configuration properties
 
-Hibernate-infinispan module comes with default configuration in `infinispan-config.xml` that is suited for clustered use. If there's only single instance accessing the DB, you can use more performant `infinispan-config-local.xml` by setting the `hibernate.cache.infinispan.cfg` property. If you require further tuning of the cache, you can provide your own configuration. Caches that are not specified in the provided configuration will default to `infinispan-config.xml` (if the provided configuration uses clustering) or `infinispan-config-local.xml`. It is not possible to specify the configuration this way in WildFly.
+As explained above, Infinispan second-level cache provider comes with default configuration in `infinispan-config.xml` that is suited for clustered use.
+If there's only single instance accessing the DB, you can use more performant `infinispan-config-local.xml` by setting the `hibernate.cache.infinispan.cfg` property.
```

Comment:
This part is not specific to WF... it's just talking about configuration in general. The only thing that we mention about WF here is that you can't pass `cfg` when running inside WF.